docs/diagrams/bugyou.txt

```diff
@@ -1,29 +1,46 @@
+                                                                                                                                          Bugyou Controller(bugyou-plugins)
+                                                                                                           +-------------------------------------------------------------------------------------------+
+                                                                                                           |                                                                                           |
+                                                                                                           |          Plugins listen to the assigned queue and                                         |
+                                                                                                           |          consumes the messages and pushes the messages to the issue tracking tools        |
```

Comment:
Proposal: Plugins listen to their own queue, consumes the messages as they are sent and pushes the messages (the message or something else? -- Alternative: report the issue from the message) to the configured issue tracking tools